src/parse_file.py

```diff
@@ -156,6 +156,13 @@ def parse_file(f: typing.TextIO, options: Options) -> MIPSFile:
                         mips_file.new_jumptable_label(function_name)
                     else:
                         mips_file.new_function(function_name)
+                elif line.startswith('func'):
+                    # Other kind of function label.
+                    function_name: str = line[:-1]
```

Comment:
I assume this `[:-1]` is to trim off the colon. Can we use `.rstrip()` here instead?